src/libs/Navigation/AppNavigator/AuthScreens.tsx

```diff
@@ -152,7 +153,7 @@ Onyx.connect({
 
         // If the current timezone is different than the user's timezone, and their timezone is set to automatic
         // then update their timezone.
-        if (timezone?.automatic && timezone?.selected !== currentTimezone) {
+        if (!lodashIsEmpty(currentTimezone) && timezone?.automatic && timezone?.selected !== currentTimezone) {
```

Comment:
why are we using lodash? AFAIK we are moving away from lodash and insist on using Typescripts own functions. Kindly update the same?